<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>223</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Particula:</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>179</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Colo(rgb):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="velocidad_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>153</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>127</y>
       <width>64</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destino en y:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="blue_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Origen en x:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>198</y>
       <width>23</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Red:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>153</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocidad:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="origeny_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>75</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>250</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>101</y>
       <width>64</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destino en x:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="destinox_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>101</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="origenx_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>49</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_lineEdit">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>23</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>75</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Origen en y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="destinoy_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>127</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="green_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>224</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="red_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>198</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>224</y>
       <width>33</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Green:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="agregar_final_pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar al final</string>
     </property>
    </widget>
    <widget class="QPushButton" name="agregar_inicio_pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar al inicio </string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrar_pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="salida">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>201</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
